.github/Architecture.md

```diff
@@ -77,6 +77,7 @@
 |  |  |  |  |  |  |- 📂 constants:
 |  |  |  |  |  |  |  |- 📜 imageConstants.ts
 |  |  |  |  |  |  |- 📂 components:
+|  |  |  |  |  |  |  |- 📜 ImageFlow.tsx
```

Comment:
doesn't the file tree generator script build this for us?

Reply:
Yes! This is automatically generated